api_posts_v2/lib/api_posts_v2/crawler.ex

```diff
@@ -0,0 +1,141 @@
+defmodule ApiPostsV2.Crawler do
+  use Tesla
+  require Logger
+
+  defp tesla_client(),
+    do:
+      Tesla.client([
+        Tesla.Middleware.FormUrlencoded
+      ])
+
+  def scrap(url) do
+    unless url do
+      Logger.info("URL is required")
```

Comment:
Se é required, um log aqui é inútil. Além disso essa func não pode ser chamada sem argumento entao pra esse unless rodar tem que passar um `nil` explicito, nao sei se tratar aqui faz sentido